parser_library/src/workspace_manager_impl.h

```diff
@@ -16,6 +16,7 @@
 #define HLASMPLUGIN_PARSERLIBRARY_WORKSPACE_MANAGER_IMPL_H
 
 #include <algorithm>
+#include <atomic>
```

Comment:
Can be removed (at least when compiled with `msvc`)

```suggestion
```

Reply:
atomic is used in the constructor, so I'd keep it.